content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -129,6 +141,9 @@ The `info` object may have some arrangement of the following fields:
 | `validated_ledger.seq`              | Number          | The [ledger index][] of the latest validated ledger. |
 | `validation_quorum`                 | Number          | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
 | `validator_list_expires`            | String          | _(Admin only)_ Either the human readable time, in UTC, when the current validator list expires, the string `unknown` if the server has yet to load a published validator list or the string `never` if the server uses a static validator list. |
+| `counters`            | Object          | This object contains performance metrics pertaining to the RPC Calls (currently executing calls and the completed calls) and the JobQueue. It also contains the details of the nodestore like node_writes, node_reads_total, node_reads_hit, etc|
+| `current_activity`            | Object          | This field contains two arrays for `jobs` and `methods`. |
```

Comment:
This feels lacking in information. What do the contents of those arrays even represent?

Reply:
Hmm, I'm not entirely sure.
@mtrippled can you please suggest better docs for this section?